scripts/wi-kind-setup.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+
+# This script requires the following tools:
+# - azure-cli : This is used for interacting with Azure services."
```

Comment:
This is super nitpicky, but why does this line (and the next few) end with an unmatched quote `"`?